### PR TITLE
open-cluster-management/backlog#18127 cim disco

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/create_infra_env.adoc [hunk 1]
```diff
@@ -18,6 +18,7 @@ You need the following prerequisites before creating an infrastructure environme
 * A configured instance of the Central Infrastructure Management (CIM) feature on your hub cluster. See xref:../clusters/create_infra_env.adoc#enable-cim[Enabling the Central Infrastructure Management service] for the procedure.
 * Your {ocp-short} https://console.redhat.com/openshift/install/pull-secret[pull secret]. See https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/images/managing-images#using-image-pull-secrets[Using image pull secrets] for more information. 
 * Your SSH key that is in your `~/.ssh/id_rsa.pub` file, by default.
+* *Disconnected environment only:* Complete the procedure for https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/scalability_and_performance/ztp-deploying-disconnected[Deploying distributed units at scale in a disconnected environment] in the {ocp-short} documentation.
```
Comment (jelkosz) [inline clusters/create_infra_env.adoc]: While the link contains also the information which we look for, it feels a bit strange to link to a document which talks about ZTP (e.g. zero touch, not CIM) and distributed units (e.g. in telco ecosystem) and also is explaining how to install ACM itself etc...

@filanov is there a better doc which could be linked from this CIM documentation about how to configure AI in a disconnected env?

At very least, lets try to explain that the part of the documentation the user should be focusing on is the "18.10. Preparing the disconnected environment"

Comment (filanov) [reply clusters/create_infra_env.adoc]: I'm not aware of any official documentation but i agree that it should be more specific, like 
https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/scalability_and_performance/ztp-deploying-disconnected#ztp-acm-preparing-to-install-disconnected-acm_ztp-deploying-disconnected

Comment (jelkosz) [reply clusters/create_infra_env.adoc]: +1, sounds good!

@cadawson so, the whole section should be something like:
* *Disconnected environment only:* Complete the procedure for https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/scalability_and_performance/ztp-deploying-disconnected#ztp-acm-preparing-to-install-disconnected-acm_ztp-deploying-disconnected[Preparing the disconnected environment] in the {ocp-short} documentation.

Comment (cadawson) [reply clusters/create_infra_env.adoc]: Thanks @jelkosz and @filanov . I updated the link to the preparation section. That should cover it, I hope. Thanks!

**Comment by cadawson** [comment]
@ianzhang366 Can you take a look at this, as well? Thanks!

**Comment by ianzhang366** [comment]
@cadawson it looks good to me as well, thank you!

File: clusters/create_infra_env.adoc (hunk 1 above)
Comment (jelkosz) [reply clusters/create_infra_env.adoc]: looks good!

**Comment by jelkosz** [comment]
lgtm

**Pull request merged**